recipes/input4mips-validation/meta.yaml

```
@@ -0,0 +1,96 @@
+{% set name = "input4mips-validation" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/input4mips_validation-{{ version }}.tar.gz
+  sha256: e1e31e0279621023bd0e5468eb02ba89661a9a0b685cd4b46f5dfa54634b121d
+
+build:
+  entry_points:
+    - input4mips-validation = input4mips_validation.cli:app
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.9.0,<4.0.0
+    - loguru >=0.7
+    - cf_xarray >=0.8
+    - xarray >=2022.0
+    - pint-xarray >=0.3
+    - numpy >=1.23,<2.0
+    - cftime >=1.6
+    - typing-extensions >=4.0
+    - pooch >=1.0
+    - netcdf4 >=1.0
+    - attrs >=22.0
+    - typer >=0.9.0,<0.10.0
+    - validators >=0.28.3,<0.29.0
+    - cattrs >=23.2.3,<24.0.0
+  run_constrained:
+    - matplotlib-base >=3.0
+    - notebook >=6.0
+
+test:
+  imports:
+    - input4mips_validation
+  commands:
+    - pip check
+    - input4mips-validation --help
+  requires:
+    - pip
+
+outputs:
+  - name: input4mips-validation
+
+  # A locked version, which will help avoid installation headaches
+  # for those who just want the CLI.
+  - name: input4mips-validation-locked
+    requirements:
+      host:
+        - python >=3.9,<4.0
+        - poetry-core
+        - pip
+      run:
+        - python ==3.9
```

```suggestion
        - python 3.9.*
```

I doubt you only want `3.9.0` but not `3.9.1` etc.

---

Thank you, very helpful!